File: VB/NonPersistentDetailViewNavigationExample.Module/BusinessObjects/IssueStatictics.vb
Imports System
Imports System.Collections.Generic
Imports DevExpress.Persistent.Base
Imports DevExpress.Xpo
Imports DevExpress.ExpressApp.DC

Namespace NonPersistentDetailViewNavigationExample.Module.BusinessObjects
    <DomainComponent, NavigationItem("Issues"), ImageName("BO_Report")> _
    Public Class IssueStatistics
        Public Property ActiveIssuesCount() As Integer
        Public Property ClosedIssuesCount() As Integer
        Public Property TotalIssuesCount() As Integer
    End Class
End Namespace

File: CS/NonPersistentDetailViewNavigationExample.Module/BusinessObjects/IssueStatictics.cs
using System;
using System.Collections.Generic;
using DevExpress.Persistent.Base;
using DevExpress.Xpo;
using DevExpress.ExpressApp.DC;

namespace NonPersistentDetailViewNavigationExample.Module.BusinessObjects {
    [DomainComponent]
    [NavigationItem("Issues"), ImageName("BO_Report")]
    public class IssueStatistics {
        public int ActiveIssuesCount { get; set; }
        public int ClosedIssuesCount { get; set; }
        public int TotalIssuesCount { get; set; }
    }
}
